### ISSUE TITLE
List Projects: is_completed should be 0 or 1

### ISSUE THREAD

**Issue by RayChenLDL** [rundef/node-testrail-api#25]
If we are using the function below to get all the projects, we have to pass 'is_completed: false' to FILTERS. But I could not retrieve any project with it.

testrail.getProjects(/*FILTERS=*/{is_completed: false}, function (err, response, projects) {
  console.log(projects);
});

While if we look at TestRail API v2 reference, is_completed is either 0 or 1, not true or false. Can you please check?

http://docs.gurock.com/testrail-api2/reference-projects

# All active projects
GET index.php?/api/v2/get_projects&is_completed=0

**Comment by djom20** [comment]
I'm facing the same issue

**Comment by djom202** [comment]
I found this code into the test and it's working fine for me:

```javascript
it('Get all - filtered', function (done) {
    testrail.getRuns(1, { is_completed: 0, suite_id: 4 }, function (err, response, body) {
      expect(err).to.be.null;
      expect(response).to.be.an.object;
      expect(body).to.be.an.array;
      done();
    });
}
```

**Comment by rundef** [comment]
Fixed in version 1.3.6
